EIPS/eip-7488.md

```diff
@@ -0,0 +1,51 @@
+---
+eip: 7488
+title: Ground Zero
+description: Implements protocol and governance changes required by the community
+author: ProphetXZ (@ProphetXZ)
+discussions-to: https://ethereum-magicians.org/t/eip-ground-zero/15480
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-06-30
+---
+
+## Abstract
+
+This EIP "Ground Zero" suggests to switch mainnet to permissioned to lower gas prices, transaction costs, and improve privacy. The adjustment requires protcol and governance changes. The EIP idea is braking and stopping the IPFS and Swarm store and broadcasting and instead suggests scalling down ethereum ressources, so that the community may be served in a more **sustainable way** in the future. In addition, to mitigate the currently existing problem with increasingly high gas prices, ethereum would benefit from switching to a permissioned mainnet (after successfully deploying a testnet version). This would greatly lower the number of requests and decrease the load for operating nodes. A further improvement would be to remove legacy APIs, so nodes no longer answer unnecessary public API requests. 
+
+## Motivation
+
+Since its humble beginnings, the ethereum project has been open to the public and seen many contributions. The past years, however, have made it clear that such an open ecosystem design leads to a wide array of problems such as ongoing consensus problems, increasingly high transaction costs, cases of fraud, false ownership claims of assets such as the DAO hack and resulting fork (see [EIP-779](./eip-779.md)), illegal contracts and privacy intrusion, data theft, artifical and malicious content, and many more such things. 
```

Comment:
- There's a typo. Ethereum should be capitalized.
- What are the consensus problems? You never specify.
- I can't understand what you mean by "false ownership claims of assets." Could you rephrase that a little better?
- The DAO hack happened in 2016.
- Who defines what contracts are illegal?
- Are there any specific cases of privacy intrusions that you know about? The only thing I can think of is that the blockchain is completely public, and transitioning to a permissioned system doesn't change that.
- You already said privacy intrusions. Data theft is wholly covered at this point.
- What is "artificial content?"